easybuild/tools/modules.py

```diff
@@ -588,7 +588,7 @@ def modpath_extensions_for(self, mod_names):
         modpath_exts = {}
         for mod_name in mod_names:
             modtxt = self.read_module_file(mod_name)
-            useregex = re.compile(r"^\s*module\s+use\s+(\S+)", re.M)
+            useregex = re.compile(r'^\s*module\s+use\s+"?([^" \t\n\r\f\v]+)"?', re.M)
```

Comment:
@boegel: I wasn't aware that one can use character classes within character classes.  But makes sense.